package.json

```diff
@@ -54,6 +54,7 @@
     "memize": "1.1.0",
     "qrcode.react": "3.1.0",
     "react-animate-height": "2.0.23",
+    "rememo": "4.0.2",
```

Comment:
`rememo` is exported from `@wordpress/data`, but it was exported recently and doesn't work on WordPress 6.4, so we need to install it until it's available in all of our supported versions of WP.